taskt/UI/Forms/frmScriptEngine.cs

```diff
@@ -235,7 +254,12 @@ private void Engine_ScriptFinishedEvent(object sender, ScriptFinishedEventArgs e
 
             Result = EngineInstance.TasktResult;
             AddStatus("Total Execution Time: " + e.ExecutionTime.ToString());
-
+            try
+            {
+                UpdateLineNumber(0);
+            }
+            catch(Exception) { }
```

Comment:
Looks like we are suppressing potential exceptions here. Wouldn't we want to log/take not of anything if this fails?